app.js

```diff
@@ -127,9 +127,20 @@ if (Configuration.get('FEATURE_MORGAN_LOG_ENABLED')) {
 app.use(bodyParser.json({ limit: '10mb' }));
 app.use(bodyParser.urlencoded({ extended: true, limit: '10mb' }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, `build/${themeName}`)));
+
 app.use('/locales', express.static(path.join(__dirname, 'locales')));
```

Comment:
no caching for locales?

Reply:
locales are also cached, worked out of the box.